config/dotenv.template

```diff
@@ -1,7 +1,12 @@
+HACKTOBERFEST_DATABASE_HOST=database
```

Comment:
So I don't think it makes sense to have the Docker-specific service values added to the `dotenv` template. I had tried to keep this separate from the Docker-specific `.env` file instructions (as opposed to having users copy the `dotenv` template over to a different file and fill in default values for services), as I thought that would be the less confusing way to keep the two workflows (non-containerized, containerized) separate. But maybe it was more confusing to have these two files live separately without explanation?
Either way, though, we should remove the Docker stuff.

Reply:
Makes sense, have removed the docker-specific stuff from here. Forgot that this can still be run w/o docker.